CSS/style.css

```diff
@@ -0,0 +1,574 @@
+/* import fonts from font.css to style.css */
+
+@import url(./fonts.css);
+
+html {
+  width: 100%;
+  align-items: center;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  font-family: 'lato', sans-serif;
+  box-sizing: border-box;
+  list-style: none;
+  text-decoration: none;
+}
+
+.body {
+  font-family: 'lato', sans-serif;
+  width: 100%;
+}
+
+.add-color {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  padding-left: 0;
+}
+
+.desk {
+  display: none;
+}
+
+.get-intouch {
+  display: none;
+}
+
+.get-intouch a {
+  list-style: none;
+  color: aliceblue;
+  font-family: 'lato', sans-serif;
+  font-size: 14px;
+  text-decoration: none;
+}
+
+.bar {
+  display: block;
+  width: 25px;
+  height: 3px;
+  background-color: black;
+  margin: 5px auto;
+  -webkit-transition: all 0.1s ease-in-out;
+  transition: all 0.1s ease-in-out;
+}
+
+.hamburger {
+  display: block;
+  margin-right: 100%;
+  padding-left: 20px;
+}
+
+.main-menu {
+  position: absolute;
+  width: 96%;
+  height: 500px;
+  display: flex;
```

Comment:
- [ ] Please kindly readjust the width of your nav links. on the provided design the nav links take only the right part of the white nav bar